keyboards/minidox/keymaps/djuggler/keymap.c

```diff
@@ -0,0 +1,200 @@
+#include "minidox.h"
+#include "action_layer.h"
```

Comment:
Could you remove all of these additional includes?  

They get included automatically, at various points in the built process, so they are redundant and not necessary here.